.github/workflows/good-first-issue.yml

```diff
@@ -15,8 +15,19 @@ jobs:
         with:
           issue-number: ${{ github.event.issue.number }}
           body: |
-            Please see [our Contributing guide](https://matplotlib.org/stable/devel/index) for
-            more information on how to contribute.
-            **We do not assign issues**, and unless there is a PR for this issue already, feel free
-            to start working on it. Do not hesitate to ask if something needs clarification - see
-            also [our different communication channels](https://matplotlib.org/stable/devel/contributing.html#get-connected).
+            *Good first issue - notes for new contributors*
+
+            This issue is suited to new contributors because it does not require
+            deep understanding of the Matplotlib internals. To get started,
+            please see our [contributing guide](https://matplotlib.org/stable/devel/index)
+
+
+            **We do not assign issues**. If there is no open pull request, feel
+            free to start working on it. If there are open PRs, please
+            collaborate on the work by reviewing it rather than
+            duplicating it in a competeing PR. Linked open PRs should be listed
+            in the issue timeline or sidebar.
```

Comment:
```suggestion
            **We do not assign issues**. Check the *Development* section in
            the sidebar for linked pull requests (PRs). If there are none, feel
            free to start working on it. If there are open PRs, please
            collaborate on the work by reviewing it rather than
            duplicating it in a competeing PR.
```